src/ocean/io/device/File.d

```diff
@@ -185,8 +185,13 @@ class File : Device, Device.Seek, Device.Truncate
 
                 if (msg.length == 0)
                 {
+                    static if (__VERSION__ < 2070)
```

Comment:
Just checked, `static if` is not necessary here, can be simply changed to `import core.stdc.string` unconditionally. Has to be at top level though, because D1 does not support imports inside function scope.